cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -145,8 +145,8 @@ private void addStepsFromTable(PhaseTapChangerAdder ptca) {
         Comparator<PropertyBag> byStep = Comparator.comparingInt((PropertyBag p) -> p.asInt("step"));
         table.sort(byStep);
         for (PropertyBag point : table) {
-            double alpha = point.asDouble("angle");
-            double rho = point.asDouble("ratio", 1.0);
+            double alpha = -point.asDouble("angle"); // CGMES and IIDM conventions are opposed for alpha
```

Comment:
Rephrase to comment for both lines stating something like:
CGMES uses `ratio` to define the relationship between voltage ends, while IIDM uses `rho`.
Ratio and rho as complex numbers are reciprocals. Given `V1` and `V2` the complex voltages at end 1 and end 2 of a branch:
```
V2 = V1 * rho
V2 = V1 / ratio
```

Reply:
This is of course equivalent to the explanation given at the beginning of the pull request, we are just introducing the word `ratio` following the naming convention in the literature.

Reply:
Done!